### DISCUSSION TITLE
Interactive maps and charts are not showing

### DISCUSSION THREAD

**Discussion by shahab-valaei** [mesa/mesa-geo#114] [Q&A] [answered]
Hello, I have a problem with the mesa-geo examples, as when I try to run the examples, only the sliders are shown and the interactive maps and charts are not showing (see the attached image, which is for geo-schelling example). Does anyone have any idea how I can resolve this issue? 
Just as a note, I don't have this issue with mesa examples, and they are run properly.

Thanks!

<img width="1505" alt="Screen Shot 2022-10-17 at 11 07 44 AM" src="https://user-images.githubusercontent.com/61154430/196217023-68e7b5d5-d271-403a-bad2-cc718ad49f9d.png">

**Comment by wang-boyu** [comment]
If you cloned Mesa-Geo and installed it via pip, could you try to install the local version instead? That is, in you cloned Mesa-Geo directory, run `pip install .` instead of `pip install mesa-geo`.

I'm guessing that the examples are looking for the mesa-geo from local folder instead of the pip-installed version, and the Leaflet js/css files are not included in the source file.

@rht Any thoughts?

**Comment by wang-boyu** [comment, marked as answer]
Oh I get it - https://github.com/projectmesa/mesa-geo/pull/109 is merged but not yet released, so there is a mismatch between the source code and the released version.

In the source code the examples are using `ModulerServer` from mesa:

```python
server = mesa.visualization.ModularServer(
    GeoSchelling, [map_element, happy_element, happy_chart], "Schelling", model_params
)
```

But this change (https://github.com/projectmesa/mesa-geo/issues/112) is not yet released to PyPI.

So to make things in sync, either:

1. In the cloned repo, do `pip install .`, or
2. In the cloned repo, do `git checkout v0.3.0` and `pip install mesa-geo`.

**Reply by shahab-valaei** [reply]
Awesome! I installed the local version using ```pip install .```, and it's working. Thanks!
